specs.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+# Specifications
```

Comment:
Strictly speaking, not all docs in this list are specifications. How about "Specifications and Design Documents"?

Reply:
Maybe, although I'm not sure we care about strict / detailed definition in the title.

Otherwise we'll need to update the top button which currently just says "Specs" too, and we don't have too much space there.

Reply:
I think for most users "Specifications" is a generic enough cover word to include design documents too; beyond that, it seems to be a nuanced detail which is already expressed in a separate list item.

Reply:
As mentioned in another thread, I moved design docs down in the list, so I think mentioning them in the title might be an unnecessary distraction.